cibuildwheel/resources/build-platforms.toml

```diff
@@ -87,6 +87,8 @@ python_configurations = [
   { identifier = "cp39-win_amd64", version = "3.9.8", arch = "64" },
   { identifier = "cp310-win32", version = "3.10.0", arch = "32" },
   { identifier = "cp310-win_amd64", version = "3.10.0", arch = "64" },
+  { identifier = "cp310-win_arm64", version = "3.9.9", arch = "ARM64" },
```

Comment:
This identifier appears to be wrong? should be `cp39...` i think.

Reply:
Shouldn't we have two, possibly, then? 3.9.9 and 3.10.0? Not sure what's actually available.

Reply:
> This identifier appears to be wrong? should be `cp39...` i think.

Ah, copy-paste error. I will fix it.

Reply:
> Shouldn't we have two, possibly, then? 3.9.9 and 3.10.0? Not sure what's actually available.

Yes, both are available. I thought I had it in my patch. Will fix thtat